### PR TITLE
community-validator: improve msgs raised

### PR DESCRIPTION
**Pull request by camilamacedo86**

### CODE REVIEW

File: pkg/validation/internal/community.go [hunk 1]
```diff
@@ -256,10 +252,13 @@ func validateImageFile(checks CommunityOperatorChecks, deprecatedAPImsg string)
 						}
 
 						if verParsed.GE(semVerOCPV1beta1Unsupported) {
-							checks.errs = append(checks.errs, fmt.Errorf("this bundle is %s. Migrate the API(s) "+
-								"for %s or use the OCP labels for compatible version(s). (e.g. LABEL %s='=v4.8')",
+							checks.errs = append(checks.errs, fmt.Errorf("this bundle is %s. Migrate the API(s) for "+
+								"%s or provide compatible version(s) by using the %s annotation in " +
+								"`metadata/annotations.yaml` to ensure that the index image will be geneared " +
```
Comment (njhale) [inline pkg/validation/internal/community.go:257]:
```suggestion
								"`metadata/annotations.yaml` to ensure that the index image will be generated " +
```

File: pkg/validation/internal/community.go [hunk 2]
```diff
@@ -256,10 +252,13 @@ func validateImageFile(checks CommunityOperatorChecks, deprecatedAPImsg string)
 						}
 
 						if verParsed.GE(semVerOCPV1beta1Unsupported) {
-							checks.errs = append(checks.errs, fmt.Errorf("this bundle is %s. Migrate the API(s) "+
-								"for %s or use the OCP labels for compatible version(s). (e.g. LABEL %s='=v4.8')",
+							checks.errs = append(checks.errs, fmt.Errorf("this bundle is %s. Migrate the API(s) for "+
```
Comment (njhale) [inline pkg/validation/internal/community.go:255]: nit: I know there's precedent in this file, but I would like to stop using string concatenation as a way of making literals more readable for engineers -- almost all text editors have line-wrapping as a standard.

File: pkg/validation/internal/community.go [hunk 3]
```diff
@@ -256,10 +252,13 @@ func validateImageFile(checks CommunityOperatorChecks, deprecatedAPImsg string)
...
+								"%s or provide compatible version(s) by using the %s annotation in " +
+								"`metadata/annotations.yaml` to ensure that the index image will be geneared " +
+								"with its label. (e.g. LABEL %s='4.6-4.8')",
 								k8sApiDeprecatedInfo,
 								deprecatedAPImsg,
+								ocpLabelindex,
```
Comment (njhale) [inline pkg/validation/internal/community.go:261]: is there supposed to be two occurrences of `ocpLabelindex` here?

Comment (camilamacedo86) [reply pkg/validation/internal/community.go:259]: 1 - using the %s annotation
2 - LABEL %s='4.6-4.8'

File: pkg/validation/internal/community.go [hunk 4]
```diff
@@ -179,15 +175,14 @@ func checkMaxOpenShiftVersion(checks CommunityOperatorChecks, v1beta1MsgForResou
 // checkOCPLabels will ensure that OCP labels are set and with a ocp target < 4.9
 func checkOCPLabelsWithHasDeprecatedAPIs(checks CommunityOperatorChecks, deprecatedAPImsg string) CommunityOperatorChecks {
 	// Note that we cannot make mandatory because the package format still valid
-	if len(checks.indexImagePath) == 0 && len(checks.indexImage) == 0 {
+	if len(checks.indexImagePath) == 0 {
```
Comment (camilamacedo86) [inline pkg/validation/internal/community.go:178]: cleanup - not used.

**Review by dinhxuanvu — APPROVED** [review]
/lgtm

**Pull request merged**